feature.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+
+interface EnvironmentVariables {
+  PORT: number;
+  TIMEOUT: string;
+}
+
+@Module({
+    imports: [ConfigModule],
+  })
+export class FeatureModule {
+    constructor(private configService: ConfigService<EnvironmentVariables>) {
+    const port = this.configService.get('PORT', { infer: true });
+    const url = this.configService.get('URL', { infer: true });
+    }
+    const dbUser = this.configService.get<string>('DATABASE_USER')
+    const dbPassword = this.configService.get<string>('DATABASE_PASSWORD');
```

Comment:
is this just an example of how to use the config service?

Reply:
Yeah, I need to make it more relevant to the project